src/game/client/gameclient.h

```diff
@@ -96,6 +96,7 @@ class CGameInfo
 	bool m_HudDDRace;
 
 	bool m_NoWeakHookAndBounce;
+	bool m_NinjaSkinForFreezed;
```

Comment:
```suggestion
	bool m_NinjaSkinForFrozen;
```

I think I'd prefer `m_NoSkinChangeForFrozen`.